src/api-client.ts

```diff
@@ -14,6 +14,9 @@ import {yubikey} from './yubikey.js'
 
 const netrc = new Netrc()
 
+export const ALLOWED_HEROKU_DOMAINS = ['heroku.com', 'herokai.com', 'herokuspace.com', 'herokudev.com'] as const
+export const LOCALHOST_DOMAINS = ['localhost', '127.0.0.1'] as const
```

Comment:
Consider using `Object.freeze()` to prevent writing/replacing values in the array 
```suggestion
export const LOCALHOST_DOMAINS = Object.freeze(['localhost', '127.0.0.1']) as const
```